src/backtest/metrics.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+
```

Comment:
Nice one! Here you could create a Metrics class and have the different metrics as attributes. Just to avoid returning a dictionary.